src/main/java/rs/teslaris/core/importer/utility/scopus/ScopusImportUtility.java

```diff
@@ -276,6 +276,7 @@ public record Entry(
         @JsonProperty("prism:publicationName") String publicationName,
         @JsonProperty("prism:issn") String issn,
         @JsonProperty("prism:eIssn") String eIssn,
+        @JsonProperty("prism:isbn") String isbn,
```

Comment:
Is there prism:eIsbn in the scopus response record? It is quite rare that a book have electronic and print version and two ISBNs, but it might be possible. For ISSN it is quite common to have issn and eIssn.

Reply:
I don't think there is, I double checked, only `prism:isbn`.